hadoop-hdds/docs/content/start/OnPrem.md

```diff
@@ -104,6 +104,17 @@ Here is an  example,
     </property>
 {{< /highlight >}}
 
+* **hdds.datanode.dir** Determines where on the local filesystem HDDS data will be
+stored. Defaults to dfs.datanode.data.dir if not specified.
+
+Here is an  example,
+{{< highlight xml >}}
+    <property>
+       <name>hdds.datanode.dir</name>
+       <value>/data/disk1/data</value>
+    </property>
+{{< /highlight >}}
+
```

Comment:
This section lists properties which are present in the `ozone-site.xml` config file created by `ozone genconf`.  `hdds.datanode.dir` is not such a property, so I think this section should not list it.

```suggestion
```